src/components/results/common/results-global-filter.tsx

```diff
@@ -120,10 +121,11 @@ const ResultsGlobalFilter: FunctionComponent<ResultsGlobalFilterProps> = ({
     filters = emptyArray,
 }) => {
     const intl = useIntl();
+    // @ts-expect-error: will be fixed in the next version of commons-ui
     const { translate } = useLocalizedCountries();
```

Comment:
~Because in my next PR, I will replace it by the one in commons-ui, and it will resolve the conflict in types.~

Done by someone else on main branch.

Reply:
revert